library/network/test/susefirewall_proposal_test.rb

```diff
@@ -10,6 +10,14 @@
 describe Yast::SuSEFirewallProposal do
   subject { Yast::SuSEFirewallProposal }
 
+  before :example do
+    # We shouldn't run this test if firewalld is running
+    if Yast::SuSEFirewall.is_a?(Yast::SuSEFirewalldClass)
+      skip "FirewallD backend is not supported by this test" do
+      end
+    end
+  end
```

Comment:
Sorry, but this is not unit test. Unit test really should not depend on runtime environement. It will produce unpredictable results.